iis-workflows/iis-workflows-transformers/src/main/java/eu/dnetlib/iis/workflows/transformers/spark/citationmatching/direct/SparkCitationMatchingDirectTransformer.java

```diff
@@ -0,0 +1,64 @@
+package eu.dnetlib.iis.workflows.transformers.spark.citationmatching.direct;
+
+import java.io.IOException;
+
+import org.apache.spark.SparkConf;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+
+import eu.dnetlib.iis.citationmatching.direct.schemas.DocumentMetadata;
+import eu.dnetlib.iis.common.spark.avro.SparkAvroLoader;
+import eu.dnetlib.iis.common.spark.avro.SparkAvroSaver;
+import eu.dnetlib.iis.transformers.metadatamerger.schemas.ExtractedDocumentMetadataMergedWithOriginal;
+
+public class SparkCitationMatchingDirectTransformer {
```

Comment:
OK, I think that the final class should not be named Spark... but for now it's not a big problem.